camel/utils/commons.py

```diff
@@ -247,10 +247,10 @@ def to_pascal(snake: str) -> str:
 
 def role_playing_with_function(
     task_prompt: str = ("Assume now is 2024 in the Gregorian calendar, "
-                        "estimate the current age of University of Oxford "
-                        "and then add 10 more years to this age, "
-                        "and get the current weather of the city where "
-                        "the University is located. And tell me what time "
+                        "estimate the current age of University of Oxford."
+                        ""
+                        ""
+                        " and tell me what time "
                         "zone University of Oxford is in."),
     function_list: Optional[List] = None,
```

Comment:
why motified the prompt here? Could you explain it to me? Thanks.

Reply:
Thanks a lot for your check, I am sorry that I changed this while I was running the example and forgot to restore it. I will restore it.